util/tune/current_settings.rb

```diff
@@ -40,10 +43,10 @@
 #   value = puppetserver_jruby_max_active_instances
 #
 def puppetserver_jruby_max_active_instances
-  if File.exist? PE_PUPPET_SERVER_CONF
-    line = File.open(PE_PUPPET_SERVER_CONF).grep(/max-active-instances/)[0]
-    value = line.scan(/\d+/).first unless line.nil?
-  end
+  return "NA" unless File.exist? PE_PUPPET_SERVER_CONF
+
+  line = File.open(PE_PUPPET_SERVER_CONF).grep(/max-active-instances/)[0]
+  value = line.scan(/\d+/).first unless line.nil?
```

Comment:
Should this be changed to `value = line.scan(/\d+/).last unless line.nil?` ??  The commit comment says that the last value in a duplicated list should win.  I may not understand what this line does.

Reply:
yeah, I just changed the indent of this.  I didn't really look at this part.  But it is totally wrong.  It doesn't ignore comments...  so I will have to rework it.